Example/Example/Examples/RequestScalingHeader.swift

```diff
@@ -12,55 +12,80 @@ import ScalingHeaderScrollView
 struct RequestScalingHeader: View {
 
     @Environment(\.presentationMode) var presentationMode
-    
```

Comment:
this separates @Environment (global) block of variables from private block, please return the \n